app/serializers/site_serializer.rb

```diff
@@ -213,6 +214,11 @@ def displayed_about_plugin_stat_groups
     About.displayed_plugin_stat_groups
   end
 
+  def show_welcome_topic_banner
+    # Site.show_welcome_topic_banner?(scope)
```

Comment:
This is commented out by mistake, right? 

Would be good to add a serializer spec as well.